core/test/allocation_test.jl

```diff
@@ -254,7 +246,8 @@ end
     end
```

Comment:
You don't have to mark any test in this testitem as broken. Your changes altered the behaviour of the allocation algorithm a bit, but mostly for the better since a larger flows are allocated. You can simply change the reference values to the new values.